buildSrc/src/main/groovy/tanagra.test-conventions.gradle

```diff
@@ -31,3 +29,9 @@ def jacocoTestReport = tasks.named('jacocoTestReport') {
 tasks.named('check').configure {
     dependsOn jacocoTestReport
 }
+
+tasks.register('noCloudAccessRequiredTests', Test) {
```

Comment:
Thanks for moving this to a central place. Just checking, this doesn't affect our ability to run all tagged tests with `./gradlew noCloudAccessRequiredTests` right?

Reply:
This file is included as a plugin into all other subprojects - verified that `./gradlew noCloudAccessRequiredTests` works as usual.